pages/osx/readlink.md

```diff
@@ -0,0 +1,8 @@
+# readlink
+
+> Follow symlinks and get symlink information.
+> More information: <https://www.gnu.org/software/coreutils/readlink>.
+
+- Print the actual full path to which the symlink points:
```

Comment:
```suggestion
- Print the actual full path to which the symlink points to:
```
thoughts?

Reply:
sounds good but I;m no english expert..

Reply:
`to` is already there ("... full path **to** which the ...")

Reply:
No, it's redundant.  I personally prefer the original because it's more advanced-sounding :) but it's probably less readable for non-native-English-speakers.
```suggestion
- Print the actual full path which the symlink points to:
```

Reply:
so what sounds better
1) `Print the actual full path to which the symlink points:`
2) `Print the actual full path which the symlink points to:`

I think 2 sounds better

Reply:
Yes, both works and the current version is more concise, however with 2 `to` it reads nicer and releases the formality of it which it doesn't really need.

Reply:
so should I change it to 2?

Reply:
With two `to`s it's actually grammatically incorrect, or at least inappropriately redundant.

The two are both **100% equivalent**, the only difference is the position.  I like the first because I'm a native speaker so interesting twists appeal to me, but most people reading these pages won't feel the same way and will be confused.  Please, @Itai-Nelken, apply my latest [suggestion](https://github.com/tldr-pages/tldr/pull/6104#discussion_r646960844) above :)

Reply:
when I said 2 I meant my 2nd suggestion that is the same as yours but I missed it